src/main/java/com/murgaloids/server/report/ReportController.java

```diff
@@ -0,0 +1,54 @@
+package com.murgaloids.server.report;
+
+import java.util.List;
+
+import lombok.NonNull;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.murgaloids.server.student.StudentRepository;
+
+@RestController
+@RequestMapping(path="/reports")
+public class ReportController {
+    @Autowired
+    private ReportRepository reportRepository;
+
+    @Autowired
+    private StudentRepository studentRepository;
+
+    @PostMapping("/add-report")
```

Comment:
Would it be possible to change this to /add over /add-report?

Reply:
Fixed